e2e/install.e2e.ts

```diff
@@ -182,62 +182,104 @@ const tests: [packageManager: string, config: InstallTest | RunTest][] = [
       lockfile: 'yarn.lock',
     },
   ],
+]
+
+describe.each(installTests)('%s → installs the cli and runs commands without errors', (_, config) => {
+  // TODO: Figure out why this flow is failing on Windows.
+  const npxOnWindows = platform() === 'win32' && 'run' in config
+
+  itWithMockNpmRegistry.skipIf(npxOnWindows)('runs the commands without errors', async ({ registry }) => {
+    // Install
+
+    const cwd = registry.cwd
+    await execa(...config.install, {
+      cwd,
+      env: { npm_config_registry: registry.address },
+      stdio: debug.enabled ? 'inherit' : 'ignore',
+    })
+
+    expect(
+      existsSync(path.join(cwd, config.lockfile)),
+      `Generated lock file ${config.lockfile} does not exist in ${cwd}`,
+    ).toBe(true)
+
+    const binary = path.resolve(path.join(cwd, `./node_modules/.bin/netlify${platform() === 'win32' ? '.cmd' : ''}`))
+
+    // Help
+
+    const helpOutput = (await execa(binary, ['help'], { cwd })).stdout
+
+    expect(helpOutput.trim(), `Help command does not start with '⬥ Netlify CLI'\\n\\nVERSION: ${helpOutput}`).toMatch(
+      /^⬥ Netlify CLI\n\nVERSION/,
+    )
+    expect(
+      helpOutput,
+      `Help command does not include '${config.packageName}/${pkg.version}':\n\n${helpOutput}`,
+    ).toContain(`${config.packageName}/${pkg.version}`)
+    expect(helpOutput, `Help command does not include '$ netlify [COMMAND]':\n\n${helpOutput}`).toMatch(
+      '$ netlify [COMMAND]',
+    )
+
+    // Unlink
+
+    const unlinkOutput = (await execa(binary, ['unlink'], { cwd })).stdout
+    expect(unlinkOutput, `Unlink command includes command context':\n\n${unlinkOutput}`).toContain(
+      `Run netlify link to link it`,
+    )
+  })
+})
+
+const runTests: [packageManager: string, config: RunTest][] = [
```

Comment:
Nice approach you came up with! Just partitioning into two sets of table tests makes sense.